View/media_control_panel.py

```diff
@@ -11,3 +11,17 @@ def __init__(self):
         to the panel using some layout.
         """
         super().__init__()
+        
+        # create play pause btn
+        self.playBtn = QPushButton()
```

Comment:
Since the button acts as both a play and pause button, a more accurate name for this button would be "play_pause_button".